### DISCUSSION TITLE
Should the anaconda & miniconda casks be banned from Homebrew?

### DISCUSSION THREAD

**Discussion by conduit242** [Homebrew/discussions#5483] [Casks] [answered]
### Output of `brew config`

```shell
none
```

### Output of `brew doctor`

```shell
none
```

### Description of issue

Both of these are under commercial licenses which notably are omitted from their casks. The company claims users have accepted the license by merely downloading the software without any explicit acceptance or click through. 

At a minimum, the casks should contain the license information and require acknowledgement.

### Relevant casks

anaconda, miniconda

**Comment by SMillerDev** [comment, marked as answer]
All agreement with licences in Homebrew is implicit. We don't ask you to read and agree to the GPL or Apache license either. Casks are mostly commercial pieces of software, so they'll each have a license that is up to the user to determine.

**Reply by conduit242** [reply]
I agree, that's why the license should be included in the cask. No license implies...no license and that's what you're agreeing to. Anaconda has been trying to ambush businesses, they should at least have to disclose it or not be enabled by frameworks like Homebrew.

**Reply by SMillerDev** [reply]
Casks don't have licences (with a couple of exceptions) because it's just too much work to keep track of them. 

Even for formulae, where a valid open source license is a requirement they are guesses since licences are complicated and we're not lawyers.

If you want to add a license warning to a cask, please make a pull request. But the onus is not on Homebrew to make sure the user understands what license the software is under, that's on the user.
